docs/plugins.md

```diff
@@ -75,29 +76,24 @@ Install the plugin and configure the track id.
 ```html
 <script>
   window.$docsify = {
-    ga: 'UA-XXXXX-Y'
-  }
+    ga: 'UA-XXXXX-Y',
+  };
 </script>
 <script src="//cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js"></script>
 <script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/ga.min.js"></script>
 ```
 
 Configure by `data-ga`.
 
+<!-- prettier-ignore -->
 ```html
 <script src="//cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js" data-ga="UA-XXXXX-Y"></script>
 <script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/ga.min.js"></script>
 ```
 
-## emoji
+## Emoji
 
-The default is to support parsing emoji. For example `:100:` will be parsed to :100:. But it is not precise because there is no matching non-emoji string. If you need to correctly parse the emoji string, you need install this plugin.
-
-```html
-<script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/emoji.min.js"></script>
-```
-
-?> If you don't want to parse to emoji, you can use __colon_<span>_</span> or `&#58;`. If you need to use in the title, we recommend using `&#58;`. For example, `&#58;100:`
+!> This plugin has been deprecated. Docsify is able to render all GitHub-style emoji without the use of this plugin as of v4.13.
```

Comment:
I think this should be kept and the deprecated warning moved to the front?

Reply:
I can do that. FWIW, here was my rational for removing the description and the CDN URL:

* The plugin's description didn't do a very good job of explaining if/when people should use it (see #1726 as an example). I can rewrite the description, but it seemed strange to spend time writing a better description for a plugin that we are no longer telling people to use.
* Since we are no longer recommending the plugin, I didn't see value in providing a CDN URL that will load an empty JS file.

If you still prefer to display a description and CDN URL, I am happy to restore them. I would like to update the description to something a bit more user-friendly though.

@sy-records -- Thoughts?

Reply:
> I would like to update the description to something a bit more user-friendly though.

I agree.

Reply:
@sy-records --

I have updated the description, restored the `<script>` markup, and moved the deprecation warning.